src/database.ts

```diff
@@ -1560,12 +1559,12 @@ class Database extends GrpcServiceObject {
   restore(backupPath: string): Promise<RestoreDatabaseResponse>;
   restore(
     backupPath: string,
-    options?: CallOptions
+    options?: grpc.CallOptions
```

Comment:
This and below should be type of `CallOptions` imported from `google-gax`

Reply:
Fixed everywhere